### PR TITLE
telemetry-gateway: add attributes to pubsub events

### PR DESCRIPTION
**Pull request by bobheadxi**
This might make it easier to the team to work with the incoming data, and could be useful for some degree of tagging.

## Test plan

Unit tests

### CODE REVIEW

**Review by nathan-downs — APPROVED** [review]
Amazing how quickly you cranked this change out!

Approved pending seeing the attribute show up on payloads in the topic.

File: cmd/telemetry-gateway/internal/events/events_test.go [hunk 1]
```diff
@@ -9,6 +9,7 @@ import (
 	"time"
 
 	"github.com/stretchr/testify/require"
+	"gotest.tools/assert"
```
Comment (unknwon) [inline cmd/telemetry-gateway/internal/events/events_test.go:12]: Wrong assert package? 😂 i.e. not the `github.com/stretchr/testify/assert`

Comment (bobheadxi) [reply cmd/telemetry-gateway/internal/events/events_test.go:12]: 🤦

Comment (bobheadxi) [reply cmd/telemetry-gateway/internal/events/events_test.go:12]: VSCode just picks one at random I swear

**Pull request merged**